contribs/gmf/less/map.less

```diff
@@ -102,3 +102,22 @@ button[ngeo-mobile-geolocation] {
     clear: left;
   }
 }
+
+/** Disclaimer */
+gmf-disclaimer {
+  position: absolute;
+  bottom: @half-app-margin;
+  left: @app-margin;
+  width: 30rem;
+  z-index: @above-content-index;
+
+  .alert {
+    padding: @half-app-margin @app-margin;
+    margin: 0 0@half-app-margin  0;
```

Comment:
I fixed it already. I simply didn't update the live demo.  I'll do this right away.